app/main/actions/search.js

```diff
@@ -36,15 +36,24 @@ const DEFAULT_SCOPE = {
  * @param {Function} callback Callback function that receives used search term and found results
  */
 const eachPlugin = (term, display) => {
+  const externalSettings = config.get('external') || {}
   // TODO: order results by frequency?
   Object.keys(plugins).forEach(name => {
+    const settings = externalSettings[name]
+    if (settings) {
+      Object.keys(settings).forEach(key => {
+        settings[key] = settings[key].value
```

Comment:
You mean having the setting name at the top level instead of all of them nested under `external`?